pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "tt-umd"
+version = "1.0.0"
+authors = [
+    { name = "Tenstorrent", email = "info@tenstorrent.com" }
+]
+description = "User Mode Driver for tenstorrent"
+readme = "README.md"
+license = { text = "Apache-2.0" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+urls = { homepage = "http://www.tenstorrent.com" }
```

Comment:
you likely want to convert this over to:
```
[project.urls]
"Homepage" = "http://tenstorrent.com"
"Bug Reports" = "https://github.com/tenstorrent/tt-umd/issues"
"Source" = "https://github.com/tenstorrent/tt-umd"
```

so this gets parsed / picked up more correctly